src/napari_spatialdata/utils/_utils.py

```diff
@@ -178,25 +178,16 @@ def _swap_coordinates(data: list[Any]) -> list[Any]:
     return [[(y, x) for x, y in sublist] for sublist in data]
 
 
-def _get_transform(element: SpatialElement, coordinate_system_name: str | None = None) -> NDArrayA:
-    affine: NDArrayA
-    transformations = get_transformation(element, get_all=True)
-    cs = transformations.keys().__iter__().__next__() if coordinate_system_name is None else coordinate_system_name
-    ct = transformations[cs]
-    affine = ct.to_affine_matrix(input_axes=("y", "x"), output_axes=("y", "x"))
-
+def _get_transform(element: SpatialElement, coordinate_system_name: str | None = None) -> None | NDArrayA:
     if not isinstance(element, (SpatialImage, MultiscaleSpatialImage, AnnData, DaskDataFrame, GeoDataFrame)):
         raise RuntimeError("Cannot get transform for {type(element)}")
-    # elif isinstance(element, (AnnData, DaskDataFrame, GeoDataFrame)):
-    #    return affine
-    #    from spatialdata.transformations import Affine, Sequence, MapAxis
-    #    new_affine = Sequence(
```

Comment:
Are you sure this is not needed? If so, better since I didn't like it, but I had to add this swap of coordiantes to make things work (I don't remember if in the context of displaying thigns properly, or of saving properly the annotatoins)

Reply:
hmm would be worth to double check

Reply:
At least with applying translation it wasn't

Reply:
true also checked, it works